### PR TITLE
Feature/add longer description

### PR DESCRIPTION
**Pull request by domharrington**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,28 +20,32 @@ yarn add build-docs
 ## Usage
 
 ```js
-const buildDocs = require('build-docs');
+const buildDocs = require('./');
```
Comment (mjcuva) [inline README.md]: Why did you change this?

File: README.md [hunk 2]
```diff
@@ -20,28 +20,32 @@ yarn add build-docs
...
+ * This creates a user in the database. Here you can add a
+ * full description.
+ *
  * @param {string} name Name of the user
  * @throws {ValidationError} Must provide all required fields
  * @returns {Object} The created user object
  */`;
 
-console.log(require('util').inspect(buildDocs(source), { depth: null }));
+console.log(require('util').inspect(buildDocs(source), { depth: null })); // eslint-disable-line no-console
```
Comment (mjcuva) [inline README.md]: I don't think we need eslint-disables in the readme

File: README.md [hunk 3]
```diff
@@ -83,6 +88,18 @@ Description is written as the first line of text in your block comment
  */
 ```
 
+#### `fullDescription`
+Full description is written as 2nd first line of text in your block comment
+
+```js
+/*
+ * Normal description
+ *
```
Comment (mjcuva) [inline README.md]: What happens if you don't have a blank line between the title and the full description?

File: README.md (hunk 1 above)
Comment (domharrington) [inline README.md]: Ahh, to generate this example bit i run `node example.js` and i must've forgotten to edit that bit.

File: README.md (hunk 2 above)
Comment (domharrington) [inline README.md]: Same as above for this one.
